extensions/wikia/Helios/HelperController.class.php

```diff
@@ -120,28 +118,7 @@ public function sendConfirmationEmail()
 			return;
 		}
 
-		$langCode = $this->getVal( 'langCode', 'en' );
-		$mailTemplate = $this->app->renderView(
-			'UserLogin',
-			'GeneralMail',
-			[
-				'language' => $langCode,
-				'type' => 'confirmation-email'
-			]
-		);
-
-		$lang = \Language::factory($langCode);
-		$mailStatus = (new GlobalStateWrapper(['wgLang' => $lang]))
-			->wrap(function() use ($user, $mailTemplate, $langCode) {
-				return $user->sendConfirmationMail(
-					false,
-					'ConfirmationMail',
-					'usersignup-confirmation-email',
-					true,
-					$mailTemplate,
-					$langCode
-				);
-			});
+		$mailStatus = $user->sendConfirmationMail( false, 'ConfirmationMail' );
 
```

Comment:
Would `sendConfirmationMail( false )` be also good here?
Why do we want to use the "confirmemail_body" instead of "usersignup-confirmation-email_body" message?